packages/p/polylith/pspec_x86_64.xml

```diff
@@ -9,14 +9,14 @@
         <License>EPL-1.0</License>
         <PartOf>programming.tools</PartOf>
         <Summary xml:lang="en">Clojure tool for managing polylith projects</Summary>
-        <Description xml:lang="en">The poly tool helps you manage your polylith projects which introduces the architectural concept of “service level building blocks” which can be combined like LEGO® bricks to build our services and systems.
+        <Description xml:lang="en">The poly tool helps you manage your polylith projects which introduces the architectural concept of &#x201c;service level building blocks&#x201d; which can be combined like LEGO&#xae; bricks to build our services and systems.
```

Comment:
Some sort of encoding error here. Please fix

Reply:
Hmmm, this might be a solus tooling bug then.

Reply:
Yeah this is a bug, I get the same. To get around this, please replace the non-ascii quotes with ascii quotes and delete the (registered) character

Reply:
@davidjharder quotes replaced in spec and package file.